app/controllers/menus_controller.rb

```diff
@@ -2,6 +2,6 @@ class MenusController < ApplicationController
 
   def show
     @restaurant = Restaurant.find(params[:restaurant_id])
+    @dishes = @restaurant.menu.dishes
```

Comment:
Why store this in a separate variable if you can access dishes through the @restaurant object? Is there a point in having 2 variables? (I asked a similar question in the previous PR and would appreciate if you all followed in the discussion on code reviews)